rust_dev_preview/cross_service/rest_ses/README.md

```diff
@@ -0,0 +1,155 @@
+in `configuration/local.yaml`, similar to the following:
+
+```yaml
+rds:
+  db_instance: auroraappdb
+  secret_arn: "arn:aws:secretsmanager:us-east-1:1111222233334444:secret:docexampleauroraappsecret8B-EXAMPLE-Dz2N2y"
+  cluster_arn: "arn:aws:rds:us-east-1:1111222233334444:cluster:docexampleauroraapp-docexampleauroraappclustereb7e-EXAMPLE"
+ses:
+  source: "report-account@amazondomains.com" # Replace with an email address that is registered with Amazon SES.
+```
+
+#### Run the service
+
+This example uses [Actix Web](https://actix.rs/) to host a local web server and REST service.
+With the web server running, you can send HTTP requests to the service endpoint to list, add, and update work items and to send email reports.
+
+```
+cargo run
+```
+
+### Webpage
+
+The REST service is designed to work with the item tracker plugin in the Elwing web client.
+The item tracker plugin is a JavaScript application that lets you manage work items, send requests to the REST service, and see the results.
+
+#### Run Elwing and select the item tracker
+
+1.  Run Elwing by following the instructions in the [Elwing README](/resources/clients/react/elwing/README.md).
+1.  When Elwing starts, a web browser opens and browses to http://localhost:3000/.
+1.  Run the item tracker plugin by selecting **Item Tracker** in the left navigation bar.
+1.  Follow the Elwing [Item Tracker instructions](/resources/clients/react/elwing/src/plugins/item-tracker/README.md).
+
+## Understand the example
+
+This example uses the Actix Web framework to host a local REST service and respond to HTTP requests.
+
+### Start up
+
+When the program first starts, it loads environment and configuration information from `src/configuration.rs`.
+It then prepares its actix web resources and SDK clients in `src/startup.rs`.
+These helpers are also used in `test/startup.rs` to ensure the application is running in a consistent environment in development, testing, and production.
+
+### Routing
+
+Top level routing happens when creating the HTTP server in `src/startup.rs`, with specific routes registered in `src/healthz.rs`, `src/collection.rs`, and `src/report.rs`.
+All routes are instrumented, and primarily serve as a facade between Actix's HTTP tooling and the SDK resources.
+
+```Rust
+/// Retrieve a single WorkItem, in a JSON body, specified by a URL parameter.
+#[actix_web::get("/{id}")]
+#[tracing::instrument(name = "Request Retrieve single WorkItem", skip(client))]
+async fn retrieve(
+    itemid: Path<String>,
+    client: Data<crate::client::RdsClient>,
+) -> Result<Json<crate::work_item::WorkItem>, crate::work_item::WorkItemError> {
+    crate::work_item::repository::retrieve(itemid.to_string(), &client)
+        .await
+        .map(Json)
+}
+```
+
+### Aurora Serverless repository
+
+The [repository.rs](src/work_item/repository.rs) file contains functions that get and set data in an Aurora Serverless database by using an Amazon RDS Data Service client.
```

Comment:
If "Amazon RDS" is a reference to the service and not part of the actual name of the client, it's "Amazon Relational Database Service (Amazon RDS)" on first mention.